roles/pulp/defaults/main.yml

```diff
@@ -30,6 +30,8 @@ pulp_user_id:
 pulp_group: pulp
 pulp_group_id:
 pulp_user_home: '/var/lib/pulp'
+pulp_static_url: 'assets'
```

Comment:
I can see this default making sense although I think it's `/assets/`.

The one below though I don't think the installer needs to know about. Unless specifically set by the user `pulp_webserver_static_dir` would have it's default from pulpcore used so I expected the installer wouldn't carry a default.

Reply:
Unfortunately, it's used here:

https://github.com/pulp/pulp_installer/blob/4fabb2cd8306d71a2cfcc3d7352644459a7da1fc/roles/pulp-webserver/templates/nginx.conf.j2#L46

https://github.com/pulp/pulp_installer/blob/4fabb2cd8306d71a2cfcc3d7352644459a7da1fc/roles/pulp-webserver/templates/pulp-vhost.conf.j2#L8

(And other places in those 2 files.)

Is it possible for the installer to run a python command to obtain the value?

The pulp-webserver role is run very late in the install; after the packages are installed.

Reply:
it is possible to get these values from dynaconf:
```
[vagrant@pulp3-source-fedora30 ~]$ dynaconf list | grep STATIC_ROOT | awk '{ print $2 }'
'/var/lib/pulp/assets/'
```